test/extensions/clusters/aggregate/cluster_test.cc

```diff
@@ -217,7 +217,16 @@ TEST_F(AggregateClusterTest, AllHostAreUnhealthyTest) {
   EXPECT_CALL(primary_load_balancer_, chooseHost(_)).WillRepeatedly(Return(host));
   EXPECT_CALL(secondary_load_balancer_, chooseHost(_)).WillRepeatedly(Return(nullptr));
 
-  for (int i = 0; i < 100; ++i) {
```

Comment:
mind including a comment here about whats going on? its not super obvious from reading the code

Reply:
Comments were added.